StyleCop.Analyzers/StyleCopTester/Program.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace StyleCopTester
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Reflection;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Microsoft.CodeAnalysis.MSBuild;
+
+    /// <summary>
+    /// StyleCopTester is a tool that will analyze a solution, find diagnostics in it and will print out the number of
+    /// diagnostics it could find. This is useful to easily test performance without having the overhead of visual
+    /// studio running.
+    /// </summary>
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            // A valid call must have at least two parameters. A solution file and /all or one /id:SAXXXX
```

Comment:
I don't do this right now. I would have to do this manually. Im going to do that